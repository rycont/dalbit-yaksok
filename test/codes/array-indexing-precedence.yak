이름 = ["김재영", "윤덕영"]
"나는" + 이름[0 + 1] 보여주기