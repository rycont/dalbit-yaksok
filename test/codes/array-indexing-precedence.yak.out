나는윤덕영